KEEP/libconfig-patch.patch

```diff
@@ -0,0 +1,10 @@
+--- libconfig-0.2/src/kconfigdata.h
++++ libconfig-0.2/src/kconfigdata.h
+@@ -24,6 +24,7 @@
+ 
+ #ifndef _KCONFIGDATA_H
+ #define _KCONFIGDATA_H
++#include <string.h>
```

Comment:
it seems this patch is not necessary. string.h is already included - see 2 lines further down

Reply:
I'm not sure why it wants explicitly <string.h> rather than <string>, but it fails to compile without the patch, and succeeds *with* the patch.

Reply:
oh, my fault, i didnt notice that 2 lines further down its just `<string>` without .h - e.g. C++ stuff

Reply:
anyway, what's the error without the patch ?